test/smoke/src/sql/areas/notebook/notebook.test.ts

```diff
@@ -143,9 +144,47 @@ export function setup(opts: minimist.ParsedArgs) {
 				await app.workbench.sqlNotebook.waitForTrustedElements();
 			});
 		});
+
+		describe('markdown', function () {
+			it('can create http link from markdown', async function () {
+				const app = this.app as Application;
+				const markdownString = '[Microsoft homepage](http://www.microsoft.com)';
+				const linkSelector = '.notebook-cell.active .notebook-text a[href=\'http://www.microsoft.com\']';
+				await verifyElementRendered(app, markdownString, linkSelector);
+			});
+			it('can create img from markdown', async function () {
+				const app = this.app as Application;
+				const markdownString = '![Churn-Index](https://www.ngdata.com/wp-content/uploads/2016/05/churn.jpg)';
+				// Verify image with the correct src and alt attributes is created
+				const imgSelector = '.notebook-cell.active .notebook-text img[src=\'https://www.ngdata.com/wp-content/uploads/2016/05/churn.jpg\'][alt=\'Churn-Index\']';
+				await verifyElementRendered(app, markdownString, imgSelector);
+			});
+		});
 	});
 }
 
+/**
+ * Verifies that the given markdown string is rendered into the expected HTML element in split view, rich text view
+ * and the text cell outside of edit mode.
+ * @param app The application
+ * @param markdownString The markdown text to enter for the element to render
+ * @param element The query selector for the element that is expected to be rendered
+ */
+async function verifyElementRendered(app: Application, markdownString: string, element: string): Promise<void> {
+	await app.workbench.sqlNotebook.newUntitledNotebook();
+	await app.workbench.sqlNotebook.addCell('markdown');
+	await app.workbench.sqlNotebook.textCellToolbar.changeTextCellView('Split View');
+	await app.workbench.sqlNotebook.waitForTypeInEditor(markdownString);
+	// Verify link is shown in split view
+	await app.code.waitForElement(element);
+	await app.workbench.sqlNotebook.textCellToolbar.changeTextCellView('Rich Text View');
```

Comment:
Would it be worth it to add a case for changing to the standalone Markdown view, just for completeness?

Reply:
Given that we've already entered the text into the markdown editor and the rendered elements are verified that doesn't really seem to be something we need to check. This test is primarily focused on ensuring that the content is rendered correctly - so if we wanted to do something with verifying that the correct markdown text is entered that should probably go in its own separate test.